src/formpack/schema/fields.py

```diff
@@ -122,7 +122,10 @@ def _get_label(self, lang=UNSPECIFIED_TRANSLATION, group_sep='/',
                     path.append(level.name)
             return group_sep.join(path)
 
-        return self.labels.get(lang, self.name)
+        if lang is None:
+            return self.name
```

Comment:
I think there actually are labels whose `lang is None`. Take a look at https://github.com/kobotoolbox/formpack/blob/34496f7ab6e409cc2c2bd569c61b053be6ef2862/src/formpack/constants.py#L14-L23

I noticed this PR is failing tests because some exports are including names where they should include labels. Maybe this is the cause.